src/packages/BalancedJs/contracts/Baln.ts

```diff
@@ -56,4 +56,29 @@ export default class Baln extends Contract {
     console.log(payload);
     return this.callIconex(payload);
   }
+
+  public async transfer(toAddress: string, value: number): Promise<any> {
```

Comment:
I'd like to use the names of the same parameters as names of smart contract method params. 
toAddress -> to